setup.py

```diff
@@ -1,15 +1,20 @@
 import os
 from setuptools import setup
+import sys
 
 with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
     README = readme.read()
 
+tests_require = ['responses>=0.5']
+if sys.version_info < (3, 3):
+    tests_require.append('mock>=1.3')
+
 # allow setup.py to be run from any path
 os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
 
 setup(
     name='django-moj-irat',
-    version='0.2',
+    version='0.3',
```

Comment:
@ushkarev A little presumptuous of me, but I figured a version bump would be in order, too…